database/migrations/sqls/20240715163245-initial-db-up.sql

```diff
@@ -0,0 +1,77 @@
+DROP TABLE IF EXISTS users CASCADE;
+DROP TABLE IF EXISTS admins;
+DROP TABLE IF EXISTS bidder CASCADE;
+DROP TABLE IF EXISTS skill CASCADE;
+DROP TABLE IF EXISTS contractor CASCADE;
+DROP TABLE IF EXISTS bids CASCADE;
+DROP TABLE IF EXISTS tender CASCADE;
+DROP TABLE IF EXISTS tender_skill CASCADE;
+DROP TABLE IF EXISTS bidder_skill CASCADE;
+
+
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(120) NOT NULL UNIQUE,
+    password_hash VARCHAR(60) NOT NULL,
+    user_type VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE admins (
+    id SERIAL PRIMARY KEY,
```

Comment:
This should be `user_id` and shouldn't be serial, it should be the same as the id in the users table